src/components/StatisticBlock.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright 2018 - 2022 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { FC, ReactNode } from 'react'
+import styled from 'styled-components'
+
+import LoadingSpinner from './LoadingSpinner'
+
+interface Props {
+  title: string
+  primary: ReactNode
+  secondary: ReactNode
+  isLoading: boolean
+}
+
+const StatisticBlock: FC<Props> = ({ title, primary, secondary, isLoading }) => {
+  return (
+    <>
+      {isLoading && (
+        <BlockCenteredContent>
+          <LoadingSpinner />
+        </BlockCenteredContent>
+      )}
+      {!isLoading && (
+        <Block>
+          <Title>{title}</Title>
+          <Primary>{primary}</Primary>
+          <Secondary>{secondary}</Secondary>
+        </Block>
+      )}
+    </>
+  )
+}
```

Comment:
Nit: I would personally do it this way:
1. Remove `FC` since we don't need `children` (same for other components in other files)
2. Remove `return` keyword since it's a dumb component
3. Use a ternary operator to avoid using a fragment

```suggestion
const StatisticBlock = ({ title, primary, secondary, isLoading }: Props) =>
  isLoading ? (
    <BlockCenteredContent>
      <LoadingSpinner />
    </BlockCenteredContent>
  ) : (
    <Block>
      <Title>{title}</Title>
      <Primary>{primary}</Primary>
      <Secondary>{secondary}</Secondary>
    </Block>
  )
```